### ISSUE TITLE
The loss function of "LA/Baseline-RawNet2"

### ISSUE THREAD

**Issue by Amforever** [asvspoof-challenge/2021#4]
As i read from "the read document of pytorch", the "CrossEntropyLoss() function" is criterion combintion of nn.LogSoftmax() and nn.NLLLoss(). Why there is still a nn.LogSoftmax() layer in the last layer of  RawNet() and if i remove the nn.LogSoftmax() there would be Bad results.

**Comment by TakHemlata** [comment]
Hi,

Before I was not aware that CCE PyTorch includes logsoftmax() function. So I wrote externally. 

I also checked by removing log-softmax from model  definition and that's gave around similar performance (~10% EER) and 2021 ASVspoof baseline performance was 9.50%  EER. I assume this difference might be due to cuda version. 

Thanks

**Comment by Amforever** [comment]
Thanks,  I had test RawNet() on ASVspoof2019 LA dataset and achieved around 4.30% EER on Eval set.  In addiiton, I find the learning rate (lr) has great impact on the EER, when i set the lr=0.005,  the RawNet was not convergenced and got the 100% EER. 
Actually, I  still doubt the principle of   end-to-end anti-spoofing systems directly using WAV, even though the RawNet can  get pretty results, thanks.

**Comment by undefinedXD** [comment]
> Thanks, I had test RawNet() on ASVspoof2019 LA dataset and achieved around 4.30% EER on Eval set. In addiiton, I find the learning rate (lr) has great impact on the EER, when i set the lr=0.005, the RawNet was not convergenced and got the 100% EER. Actually, I still doubt the principle of end-to-end anti-spoofing systems directly using WAV, even though the RawNet can get pretty results, thanks.

I'm trying to test my model on ASVspoof 2019 LA, but I have no idea that where i can get the ground truth of eval set. Can you  tell me how i can get it ?

THANKS !

**Comment by TakHemlata** [comment]
Hi,

ASVspoof 2021 challenge keys (ground truths) are
now available on asvspoof website.

You can download ASVspoof 2021 evaluation ground truths directly from
asvspoof website : https://www.asvspoof.org/

Thanks,

Regards,
Hemlata
